ili-map-pwn31.tab

```diff
@@ -198,6 +198,7 @@ i199	00039507-s
 i200	00039705-a
 i201	00040060-s
 i202	00040189-s
+i202	00040305-s
```

Comment:
Indeed, the PWN30 synset was split into two:

```
% rg "i202\t" ili-map-pwn3*
ili-map-pwn31.tab
201:i202	00040189-s
202:i202	00040305-s

ili-map-pwn30.tab
202:i202	00040058-s
```

 WN30 00040058-s {'supine%5:00:00:passive:01', 'unresisting%5:00:00:passive:01', 'resistless%5:00:00:passive:01'}
  offering no resistance; "resistless hostages"; "No other colony showed such supine, selfish helplessness in allowing 
her own border citizens to be mercilessly harried"- Theodore Roosevelt 

 WN31 00040189-s {'unresisting%5:00:00:passive:01', 'resistless%5:00:00:passive:01'}
  offering no resistance; "resistless hostages"

 WN31 00040305-s {'supine%5:00:00:passive:01'}
  passive as a result of indolence or indifference; "No other colony showed such supine, selfish helplessness in allowing her own border citizens to be mercilessly harried"- Theodore Roosevelt


If we consider the definition only, we can say that WN30 00040058-s maps to WN31 00040189-s.  But one of the senses is now in another synset and one of the examples. There are some other cases similar to that, so let us first discuss that case, ok? @fcbond @jmccrae 

WN30 00040058-s has only one `similarTo` relation with 00039592-a. This relation was projected to  WN31 00040305-s and WN31 00040189-s which are both similarTo WN31 00039705-a. Moreover, both WN31 synsets also have an antonym relation to 00038863-a. This means they could not be differentiated by their relations in WN31 so the split is suspicious, they are indistinguishable (by their relations) in both WN31 and WN30. Yep, the glosses and examples differ, but the relations are the real WordNet criteria to define and distinguish a synset.

Reply:
BTW, the ili need not be 1-1 with PWN31, right? I am assuming that one ili can map to more than one synset in the same wordnet. So if we consider that i202 is a concept that is both 00040189-s and 00040305-s according to PWN31 , is it fine, right?

Reply:
It would seem that 00040058-s and 00040189-s are the same and should both be mapped to i202. However 00040305-s is a novel sense and will need to be assigned a new ILI

Reply:
The gist at https://gist.githubusercontent.com/ekaf/8cd78cce7005abd923c7ed2af47238e2/raw/02a4d6e0700e5a8d08bda4872feae130e2b1055b/wnsplits.py pretty prints the wordnet _splits_ dictionary from NLTK, with information about how many senses are carried over into each part of the split. With WN 3.1 it outputs this file:
[out-wnsplits.txt](https://github.com/globalwordnet/cili/files/10544620/out-wnsplits.txt), listing the 33 splits since WN 3.0. The first line is:


00040058-s -> 00040305-s (1 sensekey/s) + 00040189-s (2 sensekey/s)

This shows that 00040305-s contains one sense from the source synset, while 00040189-s contains two.

Reply:
While the previous ILI mappings contained no splits, this PR introduces the following 5:

i202 00040189-s,00040305-s
i40396 00951435-n,00951878-n
i63228 07059027-n,07059160-n
i72354 06836640-n,06836790-n
i90722 10230249-n,10230422-n

So it seems that until now, mappers have made an effort to select only one most adequate target for each source. I think there is a good reason for avoiding to create splits, because having two targets yields both a true and a false positive for each involved sense.